### ISSUE TITLE
Hello, I did not find any datasets in the project. May I ask if I need to download the datasets separately and create a new dataset directory myself?

### ISSUE THREAD

**Issue by pickerinstrom** [wxjiao/UncSamp#5]
Hello, I did not find any datasets in the project. May I ask if I need to download the datasets separately and create a new dataset directory myself?

**Comment by wxjiao** [comment]
@pickerinstrom Yes, you need to obtain the datasets by yourself. Just use the official release of WMT or any translation datasets you find elsewhere. Thanks.
